src/storage/builder.rs

```diff
@@ -92,26 +92,21 @@ impl Builder {
     /// Sets blob file name prefix, e.g. if prefix set to `hellopearl`,
     /// files will be named as `hellopearl.[N].blob`.
     /// Where N - index number of file
-    /// Must be not empty
-    /// # Examples
-    /// ```no-run
-    /// let builder = Builder::new()
-    ///     .work_dir("/tmp/pearl/")
-    ///     .max_blob_size(1_000_000)
-    ///     .max_data_in_blob(1_000_000_000)
-    ///     .blob_file_name_prefix("enough");
-    /// ```
+    /// If the prefix is empty, param won't be set.
     pub fn blob_file_name_prefix<U: Into<String>>(mut self, blob_file_name_prefix: U) -> Self {
         let prefix = blob_file_name_prefix.into();
         if prefix.is_empty() {
             error!("passed empty file prefix, not set");
         } else {
             self.config.blob_file_name_prefix = Some(prefix);
-            info!(
-                "blob file format: {}.{{}}.blob",
-                self.config.blob_file_name_prefix.as_ref().unwrap()
-            );
         }
         self
     }
+
+    /// # Description
+    /// Sets key size limit
+    pub fn key_size(mut self, key_size: u64) -> Self {
```

Comment:
`u64` is to huge for key size) `u8` is too small and `u16` seams to be fine)

Reply:
Agree